src/SDKs/_metadata/cognitiveservices_data-plane_ComputerVision.txt

```diff
@@ -1,11 +1,11 @@
-2018-01-22 18:37:37 UTC
+2018-05-25 15:48:30 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
-Branch:      current
-Commit:      cb9c18b988dd67894653a34dc55b7978403b120a
+GitHub user: cthrash
```

Comment:
Please generate code from the Azure master branch, we cannot accept PRs with code generated from non-master branches